Services/Maps/css/service_openlayers.css

```diff
@@ -0,0 +1,53 @@
+.map:-moz-full-screen {
+    height: 100% !important;
+    width: 100% !important;
+}
+.map:-webkit-full-screen {
+    height: 100% !important;
+    width: 100% !important;
+}
+.map:-ms-fullscreen {
+    height: 100% !important;
+    width: 100% !important;
+}
+.map:fullscreen {
+    height: 100% !important;
+    width: 100% !important;
+}
```

Comment:
Why are these required?

Reply:
without these css snippets the size of the map didn´t change and the rest of the page is only filled black.